lambdas/search/index.py

```diff
@@ -67,14 +67,14 @@ def lambda_handler(request):
     user_size = request.args.get('size', DEFAULT_SIZE)
     user_source = request.args.get('_source', [])
     # 0-indexed starting position (for pagination)
-    user_from = request.args.get('from', 0)
+    user_from = int(request.args.get('from', 0))
```

Comment:
Does it come as `str`?

Reply:
i'm not sure, but since the invariant below is failing when `from=0`, i can deduce that it's not an `int`

Reply:
It does. For some reason, the way we do query parameter processing the values always come as strings.